imagedephi/main.py

```diff
@@ -45,21 +45,24 @@ def imagedephi(ctx: click.Context, override_rules: TextIO | None) -> None:
 @click.argument("input-path", type=click.Path(exists=True, readable=True, path_type=Path))
 @click.argument(
     "output-dir",
+    required=False,
```

Comment:
This is actually an opportunity to make the CLI take [variadic input arguments](https://click.palletsprojects.com/en/8.1.x/arguments/#variadic-arguments). This would allow it to work with shell expansion or multiple named files.

If we want that, we should make the output directory into an option, instead of an argument.

Reply:
@marySalvi Brought up the point that this change would require some additional internal logic. Let's do it in a follow-up PR.